AppCenter/AppCenter/Internals/Storage/MSLogDBStorage.m

```diff
@@ -40,27 +40,28 @@ - (instancetype)init {
 
 #pragma mark - Save logs
 
-- (BOOL)saveLog:(id<MSLog>)log withGroupId:(NSString *)groupId {
+- (BOOL)saveLog:(id<MSLog>)log withGroupId:(NSString *)groupId critical:(BOOL)critical {
```

Comment:
Can we make a wrapper for this without the `critical` parameter that always passes `critical:nil`?  It would save passing a bunch of `nil` parameters

Reply:
As discussed would rather switch to the flag directly, so an integer.